lib/namespace.js

```diff
@@ -25,10 +25,15 @@ var Namespace = createClass({
   },
 
   /*
-   * Use a namespace plugin.
+   * Use a namespace plugin or load a generic plugin.
    */
   use: function(plugin) {
-    plugin.namespace({base: this});
+    if (plugin.namespace) {
+      plugin.namespace({base: this});
+    }
+    else if (plugin.load) {
```

Comment:
What about a plugin that both provides a namespace and also wants to do something on load?

Perhaps make this two if statements instead of an `else if`.